charts/gadget/templates/daemonset.yaml

```diff
@@ -94,6 +94,8 @@ spec:
               value: "/host"
             - name: IG_EXPERIMENTAL
               value: {{ .Values.config.experimental | quote }}
+            - name: CONTAINERD_NAMESPACE
+              value: "k8s.io"
```

Comment:
containerd k8s namespace is [fixed to be "k8s.io"](https://github.com/containerd/containerd/blob/main/pkg/cri/constants/constants.go#L21) so doesn't makes sense to make it configurable when running in Kubererntes. Perhaps, it can be moved to entrypoint/Dockerfile to be clear?